java/src/jmri/server/json/JsonHttpService.java

```diff
@@ -97,7 +97,6 @@ public JsonNode doPut(@Nonnull String type, @Nonnull String name, @Nonnull JsonN
      * @param locale the requesting client's Locale
      * @throws JsonException if this method is not allowed or other error occurs
      */
-    @Nonnull
```

Comment:
Removing this is not the correct fix (the problem is void). Please revert this single fix, since #5061 is assigned to me to fix.